server/src/main/java/com/lunchtime/controllers/UserController.java

```diff
@@ -0,0 +1,33 @@
+package com.lunchtime.controllers;
+
+
+import com.lunchtime.models.Users;
+import com.lunchtime.repository.UserRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.web.bind.annotation.*;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+    private UserRepository userRepository;
+    private BCryptPasswordEncoder bCryptPasswordEncoder;
+
+    public UserController(UserRepository userRepository,
+                          BCryptPasswordEncoder bCryptPasswordEncoder) {
+        this.userRepository = userRepository;
+        this.bCryptPasswordEncoder = bCryptPasswordEncoder;
+    }
+
+    @GetMapping("/all")
```

Comment:
You don't need to use ending 'all/ in get method. You can use standard REST approach